gui/syncthing/core/eventService.js

```diff
@@ -6,6 +6,16 @@ angular.module('syncthing.core')
 
         var lastID = 0;
         var self = this;
+        
+        var shortcutIcon=$("#favicon");
+        
+        //Preloads Error icon
+        //To ensure it's available offline
+        new Image().src="/assets/img/favicon_error.png";
+        
+        function changeFavIcon (icon) {
+            shortcutIcon.attr("href","/assets/img/" + icon);
+        }
```

Comment:
maybe simply to

    shortcutIcon.attr("href","/assets/img/favicon_" + state + ".png");

since all icons have the similar file names